protocol/avdl/keybase1/identify.avdl

```diff
@@ -44,7 +44,8 @@ protocol identify {
     boolean noSkipSelf=true,
     boolean canSuppressUI=false,
     TLFIdentifyBehavior identifyBehavior=0,
-    boolean forceDisplay=false
+    boolean forceDisplay=false,
+    boolean actLoggedOut=false
```

Comment:
woah I didn't know about this syntax. What does it do?

Reply:
it's AVDL syntax that allows default values